python/opendht.pyx

```diff
@@ -57,6 +57,20 @@ cdef inline void shutdown_callback(void* user_data) with gil:
         cbs['shutdown']()
     ref.Py_DECREF(cbs)
 
+cdef inline bool get_filter(const cpp.Value& value, void *user_data) with gil:
+    cbs = <object>user_data
+    if 'filter' in cbs and cbs['filter']:
+        # Deep of const Value to Python level Value
```

Comment:
Yeah. It was the only I found to get around the fact that the C++ filter callback takes a `const Value` and the Python class `Value` has a non-const `shared_ptr[Value]`.

Indeed, instead of deep copying the value, we could call the filter from the python layer in the get callback.